create_db.py

```diff
@@ -4,5 +4,6 @@
 
 
 app = create_app("development")
-db.app = app
-db.create_all()
+with app.app_context():
+    db.app = app
+    db.create_all()
```

Comment:
``` bash
## Creating database
docker-compose run --rm web python ./create_db.py
WARN[0000] The "APPROVE_REGISTRATIONS" variable is not set. Defaulting to a blank string. 
[+] Building 0.0s (0/0)                                                                                                                                                                       
[+] Creating 1/0
 ✔ Container openoversight-postgres-1  Running                                                                                                                                           0.0s 
[+] Building 0.0s (0/0)                                                                                                                                                                       
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
/usr/local/lib/python3.8/site-packages/flask_limiter/extension.py:293: UserWarning: Using the in-memory storage for tracking rate limits as no storage was explicitly specified. This is not recommended for production use. See: https://flask-limiter.readthedocs.io#configuring-a-storage-backend for documentation about configuring the storage backend.
  warnings.warn(
[2023-06-22 20:21:00,955] INFO in __init__: OpenOversight startup
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
/usr/local/lib/python3.8/site-packages/flask_limiter/extension.py:293: UserWarning: Using the in-memory storage for tracking rate limits as no storage was explicitly specified. This is not recommended for production use. See: https://flask-limiter.readthedocs.io#configuring-a-storage-backend for documentation about configuring the storage backend.
  warnings.warn(
[2023-06-22 20:21:00,974] INFO in __init__: OpenOversight startup
Traceback (most recent call last):
  File "./create_db.py", line 8, in <module>
    db.create_all()
  File "/usr/local/lib/python3.8/site-packages/flask_sqlalchemy/extension.py", line 884, in create_all
    self._call_for_binds(bind_key, "create_all")
  File "/usr/local/lib/python3.8/site-packages/flask_sqlalchemy/extension.py", line 855, in _call_for_binds
    engine = self.engines[key]
  File "/usr/local/lib/python3.8/site-packages/flask_sqlalchemy/extension.py", line 636, in engines
    app = current_app._get_current_object()  # type: ignore[attr-defined]
  File "/usr/local/lib/python3.8/site-packages/werkzeug/local.py", line 508, in _get_current_object
    raise RuntimeError(unbound_message) from None
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
the current application. To solve this, set up an application context
with app.app_context(). See the documentation for more information.
make: *** [create_db] Error 1
```

This change was made so that I was no longer working `outside of the context`.